lamindb/models/schema.py

```diff
@@ -318,28 +394,33 @@ def __init__(
         type: Feature | None = kwargs.pop("type", None)
         is_type: bool = kwargs.pop("is_type", False)
         otype: str | None = kwargs.pop("otype", None)
-        minimal_set: bool = kwargs.pop("minimal_set", True)
```

Comment:
Please don't hard remove right away. Instead, if a user supplies this, print a warning that the setting will be ignored and that the user should instead use `Feature().with_config(optional=True)` for __all__ features.